tsconfig.json

```diff
@@ -10,7 +10,6 @@
         "strictNullChecks": true,
         "noUnusedLocals": true,
         "noUnusedParameters": false,
-        "noFallthroughCasesInSwitch": true
```

Comment:
instead of removing this, can't we just put a // fallthrough comment in the relevant places?